zhm/zhm.c

```diff
@@ -414,12 +414,13 @@ init_hm(void)
 #ifndef DEBUG
      if (!inetd && !nofork)
```

Comment:
Can you add { } to this statement? I don't like unblanaced if-else clauses where one has it and the other does not.